packages/debug/src/browser/debug-frontend-application-contribution.ts

```diff
@@ -67,6 +67,8 @@ export namespace DebugMenus {
     export const DEBUG_BREAKPOINTS = [...DEBUG, 'f_breakpoints'];
 }
 
+
+
```

Comment:
Formatting: Please remove the added lines here, they will lead to linting errors.

Reply:
done